Joserubemn31/cooking--frontend
# Create Carousel One

what i did:
- I created 4 components: CarouselOne (main component), Slider and SliderMobile (where everything about the carousel works) and SingleSlide (about the content of every slide). I thought it would be nice, because we can get elements and change the content easily to use the carousel in other places (now, everything is static - same content).
- you can see the Slider.css, that was where I costumized some things of Carousel

how to test:
- on main branch, git pull
- git checkout feature/carousel-one
- yarn start
- see if the carousel is working (web, tablet and mobile versions)

![c1](https://user-images.githubusercontent.com/61155203/104131818-ba6e8f00-5357-11eb-9c3a-7d69ef75c0a8.gif)
![c 2](https://user-images.githubusercontent.com/61155203/104131820-bc385280-5357-11eb-9dd7-1c9e59a9e228.gif)

If you have any questions, comment here! :)

resolves #9
